src/Tribe/Views/V2/Views/Widgets/Widget_View.php

```diff
@@ -61,6 +62,37 @@ protected function setup_template_vars() {
 		return $template_vars;
 	}
 
+	/**
+	 * Sets up the View repository arguments from the View context or a provided Context object.
+	 *
+	 * @since 4.9.3
+	 *
+	 * @param  Context|null $context A context to use to setup the args, or `null` to use the View Context.
+	 *
+	 * @return array The arguments, ready to be set on the View repository instance.
+	 */
+	protected function setup_repository_args( Context $context = null ) {
+		bdump('widget_setup_repository_args');
```

Comment:
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: Expected 1 spaces before closing parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: dbad249068566b809743b38392432894</sub>